cmd/spiffe-aws-assume-role/cli/cli.go

```diff
@@ -68,9 +75,19 @@ func RunWithDefaultContext(args []string) error {
 }
 
 func Run(context *CliContext, args []string) error {
+	err := doRun(context, args)
+	if err != nil {
+		logger := createLogger()
+		logger.Error(err)
+	}
+
+	return err
+}
+
+func doRun(context *CliContext, args []string) error {
```

Comment:
I wanted a single choke point to ensure logging always does the right thing, and since the original Run method had a couple exit points I thought wrapping it made sense. However... given that Go has defer, perhaps it would be better to put this logic in a defer block?